<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>351</width>
      <height>651</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>351</width>
      <height>651</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请输入合式范式</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>301</width>
        <height>531</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>216</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_open_sheet">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>E</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="minimumSize">
              <size>
               <width>94</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>(</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="minimumSize">
              <size>
               <width>95</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="minimumSize">
              <size>
               <width>94</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>!</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_11">
             <property name="minimumSize">
              <size>
               <width>94</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>|</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_12">
             <property name="minimumSize">
              <size>
               <width>95</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;&amp;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_13">
             <property name="minimumSize">
              <size>
               <width>94</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;-&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_14">
             <property name="minimumSize">
              <size>
               <width>94</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>-&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_15">
             <property name="minimumSize">
              <size>
               <width>95</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>退格</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_16">
             <property name="minimumSize">
              <size>
               <width>94</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="minimumSize">
              <size>
               <width>214</width>
               <height>0</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>308</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>退出</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="minimumSize">
          <size>
           <width>299</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2" native="true"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>347</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
